packages/firestore/src/core/firestore_client.ts

```diff
@@ -501,6 +503,15 @@ export function firestoreClientTransaction<T>(
   return deferred.promise;
 }
 
+export function firestoreClientRunAggregationQuery(
+  client: FirestoreClient,
+  query: AggregateQuery
+): Promise<AggregateQuerySnapshot> {
+  return client.asyncQueue.enqueue(async () => {
```

Comment:
Can you remove `async` from the lambda definition? I'm not sure if it's strictly required, so if it's not, let's remove it.